setup/requirements.txt

```diff
@@ -1,5 +1,3 @@
 -r requirements/requirements_base.txt
 -r requirements/requirements_prediction.txt
 -r requirements/requirements_tracking.txt
--r requirements/requirements_nuimages.txt
```

Comment:
Note that we cannot remove this here. Perhaps we can remove that line dynamically in the dockerfile?